source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.ApplyDBMigrationsApp/Scripts/Test/202201200831 Add metering points.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO [Charges].[MeteringPoint] VALUES (NEWID(), '571313180000000005', 1, 800, '2020-12-31T23:00:00', 2, 3)
+INSERT INTO [Charges].[MeteringPoint] VALUES (NEWID(), '701313180000000005', 3, 800, '2020-12-31T23:00:00', 1, 3)
+INSERT INTO [Charges].[MeteringPoint] VALUES (NEWID(), '901313180000000005', 1, 800, '2020-12-31T23:00:00', 1, 3)
```

Comment:
Right now the values doesn't pose a problem, but they probably will later, when we begin modus checking them.
I suggest to do this change now, to make them modus 11 valid. 
And Please update CreateDefaultChargeLinksReceiverTests.cs where they are used.
```suggestion
INSERT INTO [Charges].[MeteringPoint] VALUES (NEWID(), '571313180000000012', 3, 800, '2020-12-31T23:00:00', 1, 3)
INSERT INTO [Charges].[MeteringPoint] VALUES (NEWID(), '901313180000000029', 1, 800, '2020-12-31T23:00:00', 1, 3)
```

Reply:
I don't think we should ever validate these identification numbers as they are owned by the actor register. We should simply take them as-is from there. But I still agree that it's prettier to use valid numbers.